cypress/integration/projectSettings/project_settings.ts

```diff
@@ -529,4 +559,35 @@ describe("Project Settings when defaulting to repo", () => {
       cy.dataCy("alias-row").should("have.length", 0);
     });
   });
+
+  describe("Virtual Workstation page", () => {
+    before(() => {
+      cy.dataCy("navitem-virtual-workstation").click();
+    });
+
+    it("Shows repo commands", () => {
+      cy.dataCy("add-button").should("not.exist");
+      cy.dataCy("command-row").should("have.length", 2);
+      cy.dataCy("command-row").each(() => {
+        cy.get("input").should("be.disabled");
+        cy.get("textarea").should("be.disabled");
+      });
+    });
+
+    it("Allows overriding without adding a command", () => {
+      cy.dataCy("commands-override-radio-box")
+        .find("input")
+        .first()
+        .parent()
```

Comment:
i wonder does `cy.getInputByLabel` work instead?. You should be able to target the exact input element you want as long as its label is defined correctly (Which i assume it would because accessibility)